### ISSUE TITLE
Invaild facebook permission

### ISSUE THREAD

**Issue by akmittal006** [privly/privly-android#64]
"read_mailbox" is deprecated.

This app is throwing a Invalid Scope for read_mailbox.

how will app display inbox messages?

**Comment by vshivam** [comment]
Yeah, the read_mailbox permission was recently deprecated. We need to move from inbox messages to user's feed.

**Comment by vshivam** [comment]
Looks like the public_feed api has also been disabled. 

https://developers.facebook.com/docs/public_feed
